doc/languages-frameworks/bower.section.md

```diff
@@ -0,0 +1,119 @@
+# Bower {#sec-bower}
+
+[Bower](https://bower.io) is a package manager for web site front-end components. Bower packages (comprising of build artefacts and sometimes sources) are stored in `git` repositories, typically on Github. The package registry is run by the Bower team with package metadata coming from the *bower.json* file within each package.
+
+The end result of running Bower is a *bower_components* directory which can be included in the web app's build process.
```

Comment:
I think it is more common to mark paths using literals in Markdown.

Reply:
Updated